.github/workflows/docker.yml

```diff
@@ -61,7 +65,9 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           push: true
-          labels: "gitsha1=${{ github.sha }}"
+          labels: |
+            gitsha1=${{ github.sha }}
+            version=${{ env.SYNAPSE_VERSION }}
```

Comment:
It might be friendlier to use `org.opencontainers.image.version` per https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys (?)